### ISSUE TITLE
Nobita's Biohazard Korean translation tries to load file name with wrong encoding

### ISSUE THREAD

**Issue by fdelapena** [EasyRPG/Player#649]
[Game download](https://docs.google.com/file/d/0BziqGOORT_8aMm5wZ3RaMEs4SWs/edit?pli=1), note: zip contains Korean filenames, watch out when unpacking.

How to reproduce: just play the intro cutscene, it will fail in the first conversation after the Doraemon's magic door entrance. It will fail to load FaceSet/롥릐뚺딁. Log says: `Cannot find: FaceSet/[\0x0018][\0x0018]�[\0x0018]l[\0x0018]�[\0x0018]`.
(The [\0x0018] are handwritten intentionally here to prevent markdown converting these into 0xfffd).

**Comment by Ghabry** [comment]
Hard to extract correctly ;)

**Comment by fdelapena** [comment]
As Zegeri pointed in chat, `--encoding windows-949-2000` seems to convert the string properly.
However, it does not convert `\` to Won sign (keeps the `\`).

About another issue with the failure when finding `오브젝~1`, this file exists in some Korean IB game torrent after [doing a search](https://encrypted.google.com/search?q="오브젝*1.png"), apart of `오브젝트1` (available too in the same torrent).

**Comment by fdelapena** [comment]
This encoding works with the faceset and works with Won out of the box:
`ibm-1363_P110-1997`.

Note: in case to be used in static builds, it will need to be added to icudata and remove the old one (and update `reader_util.cpp` in liblcf).

**Comment by Ghabry** [comment]
Could we delay this to 0.4.1? Updating ICU everywhere takes a while...

**Comment by fdelapena** [comment]
Sure, it also depends on #673 first to test and make which is the right encoding to use yet.

**Comment by fdelapena** [comment]
I have uploaded a new icudt56l version including all KSC variants (every variant has very low kb cost). After rebuilding all toolchains with icu56 and the new icudata it should be easier to test and fix this issue for all player platforms.

**Comment by fdelapena** [comment]
Tested all available working ICU encodings for Korean with the following results:

Encoding | FaceSet fails? | Middle dot fails? | ₩ or \?
---------|----------------|-------------------|--------------
ibm-1363_P110-1997 | **No** | Yes | **₩**
ibm-1363_P11B-1998 | **No** | Yes | \
ibm-949_P110-1999 | Yes | Yes | **₩**
ibm-949_P11A-1999 | Yes | Yes | \
ibm-970_P110_P110-2006_U2 | Yes | **No** | \
windows-949-2000 | **No** | **No** | \

So the only way is windows-949-2000.
Losing the Won support is not nice but it can be worked around into a separated issue.

It's time to update icudata for all toolchains to include the following Korean mapping tables:
- `ibm-1363_P11B-1998.cnv` (windows-949-2000 relies on this one)
- `windows-949-2000.cnv`

Previously existing Korean mapping tables in custom icudata can (should) be dropped.

**Comment by Ghabry** [comment]
Lets patch ICU to fix this :+1:  ;)
